docusaurus.config.js

```diff
@@ -30,6 +30,12 @@ const config = {
         srcDark: "design/logo/svg/Mesh_Logo_White.svg",
       },
       items: [
+        {
+          label: "Blog",
+          to: "blog",
+          blogTitle: "Meshtastic Blog!",
+          blogDescription: "Placeholder",
```

Comment:
How about something like `Posts from Meshtastic community members including devs, maintainers, and those who just want to share`